### DISCUSSION TITLE
After login, I get redirected to http://localhost:8080/service-worker.js

### DISCUSSION THREAD

**Discussion by rickiewars** [filamentphp/filament#13214] [Help]
### Package

Panel builder

### Package Version

v3.2.86

### How can we help you?

On my localhost environment, after login into my panel, I get redirected to `http://localhost:8080/service-worker.js`.
I have SPA enabled on my panel and do not have any reference to this `service-worker.js` file in my project, except for one in a `storage/framework/sessions/<session>` file.
I do not have a `service-worker.js` file created so after login, I get a 404 page because the file cannot be found.

What is this service-worker.js file and why does my panel seem to need this?
What should be the content of the service-worker.js file?
I googled around a bit but couldn't find it referenced in FillamentPHP, Laravel or Livewire, except for [this discussion](https://github.com/filamentphp/filament/discussions/1856)
I only know it has something to do with SPA applications.

My thanks in advance.

**Comment by rickiewars** [comment]
A correction, I now also tried without SPA enabled and my issue remains.
I also forgot to mention that I don't have this issue in an incognito window and I use the Firefox browser.
For now, I solved my issue by clearing all data from the url in my browser but I still would like to know how this could occour.

**Comment by Ayub474** [comment]
Tidak

**Comment by Ayub474** [comment]
Semoga berhasil
